production/simple-scalable/docker-compose.yaml

```diff
@@ -0,0 +1,175 @@
+---
+version: "3"
+
+networks:
+  loki:
+
+services:
+  read:
+    image: grafana/loki:2.4.1
+    command: "-config.file=/etc/loki/config.yaml -target=read"
+    ports:
+      - 3100
+      - 7946
+      - 9095
+    volumes:
+      - ./loki-config.yaml:/etc/loki/config.yaml
+    depends_on:
+      - minio
+    networks: &loki-dns
+      loki:
+        aliases:
+          - loki
+
+  write:
+    image: grafana/loki:2.4.1
+    command: "-config.file=/etc/loki/config.yaml -target=write"
+    ports:
+      - 3100
+      - 7946
+      - 9095
+    volumes:
+      - ./loki-config.yaml:/etc/loki/config.yaml
+    depends_on:
+      - minio
+    networks:
+      <<: *loki-dns
+
+  promtail:
+    image: grafana/promtail:2.4.1
+    logging: &logging
+      driver: loki-compose
+      options:
+        loki-url: "http://localhost:3100/loki/api/v1/push"
+        loki-retries: "1"
+        loki-tenant-id: "tenant1"
+    volumes:
+      - /var/log:/var/log
+      - ./promtail-config.yaml:/etc/promtail/config.yaml:ro
+    command: -config.file=/etc/promtail/config.yaml
+    depends_on:
+      - gateway
+    networks:
+      - loki
+
+  minio:
```

Comment:
Out of curiosity, why did you opt for Minio and not local storage?

Reply:
so they can scale the instances without making any other changes.